content/en/synthetics/browser_tests/actions.md

```diff
@@ -164,6 +164,45 @@ You can run browser tests within other browser tests, up to two levels of nestin
 
 **Note**: If it does not make sense for you to run your subtest independently, you can pause it. It will continue to be called as part of your main test, but it will not be executed individually.
 
+## HTTP Requests
+
+You can run HTTP requests as part of your Browser tests. 
+
+{{< img src="synthetics/browser_tests/recorder_http_requests.png" alt="HTTP Request step"  style="width:70%;" >}}
+
+### Perform an HTTP request
+
+To define your HTTP request:
+
+1. Choose the **Method** and **URL** to query. Available methods are: `GET`, `POST`, `PATCH`, `PUT`, `HEAD`, `DELETE`, and `OPTIONS`.
+2. Optionally specify **Advanced Options**:
+     * Follow redirects: Toggle to have the monitored endpoint follow up to ten redirects.
+     * Allow insecure certificates: Toggle to have your HTTP test go on with connection even if there is an error when validating the certificate.
+     * Headers: Defined headers override the default browser headers. For example, set the User Agent in the header to [identify Datadog scripts][1].
+     * Authentication: HTTP basic authentication with username and password
+     * Body: Request body and body type (`text/plain`, `application/json`, `text/xml`, `text/html`, or `None`). **Note**: The request body is limited to a maximum size of 50 kilobytes.
+     * Cookies: Defined cookies are added to the default browser cookies. Set multiple cookies using the format `<COOKIE_NAME1>=<COOKIE_VALUE1>; <COOKIE_NAME2>=<COOKIE_VALUE2>`.
+3. Click on **Test URL** to try out your request configuration. You should see a response preview show up allowing you to confirm whether you are getting the expected response data.
+
+{{< img src="synthetics/browser_tests/http_request.png" alt="Make HTTP Request"  style="width:80%;" >}}
+
+### Add assertions
+
+You can also optionally decide to base your step success on assertions about the defined HTTP request:
+
+| Type          | Operator                                                                        | Value type                 |
+|---------------|---------------------------------------------------------------------------------|----------------------------|
+| Status Code   | `is`, `is not`                                                                  | _Integer_                  |
+| Response time | `lessThan`                                                                      | _Integer (ms)_             |
+| Headers       | `contains`, `does not contain`, `is`, `is not` <br> `matches`, `does not match` | _String_ <br> _[Regex][7]_ |
+| Body          | `contains`, `does not contain`, `is`, `is not` <br> `matches`, `does not match` | _String_ <br> _[Regex][7]_ |
+
```

Comment:
I think this table would be easier to read if the `Value type` column was in the center.

Reply:
Done!